### DISCUSSION TITLE
OWASP Top 10 or SAST evaluations in pipeline?

### DISCUSSION THREAD

**Discussion by flickerfly** [istio/istio#47724] [General]
I was looking through the github actions code and trying to determine if there is a a place some sort of SAST evaluation was happening against the Istio code before a release. I didn't see anything, but my understanding of GitHub Actions isn't as strong as others.

If this isn't in place, is it something would be considered valuable?

**Comment by howardjohn** [comment]
We do run `gosec` linter as part of our CI/CD under the "lint" job. In general, we run all our CI/CD using prow, and don't intend to us any other CI/CD platforms. If there is valuable tools to add, we can add them to the existing infrastructure; I haven't seen any that would add value, but I am not looking either so there could be some out there :slightly_smiling_face: . We also run vulnerability scans to automate our dependency updates.
